Assets/Source/Player/HUD.cs

```diff
@@ -36,7 +36,17 @@ void OnGUI()
 			StringBuilder buffer = new StringBuilder();
 
 			foreach(var obj in compie.SelectedUnits) {
-				buffer.Append(obj.transform.root.gameObject.name + "\n");
+				buffer.Append(obj.transform.root.gameObject.name);
+
+				Loyal temp = obj.transform.root.GetComponent<Loyal>();
```

Comment:
`temp` => `loyalComponent` ? Would like to use description variable names as much as possible.